### PR TITLE
[chore] v0.1.7

### PR DESCRIPTION
**Pull request by blefaudeux**
# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos, doc improvements)
- [ ] Did you read the [contributor guideline](https://github.com/facebookresearch/fairscale/blob/master/CONTRIBUTING.md)?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

## What does this PR do?
Packages a host of fixes in ShardedDDP raised by better unit testing, I think that it's worth it for many users who are not tracking master but installing via pip
Fixes https://github.com/facebookresearch/fairscale/issues/395 and https://github.com/facebookresearch/fairscale/issues/397 by adding the existing fix to the pip package eventually

## PR review
Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by min-xu-ai — APPROVED** [review]

File: fairscale/__init__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # LICENSE file in the root directory of this source tree.
 
 # Please update the doc version in docs/source/conf.py as well.
```
Comment (min-xu-ai) [inline fairscale/__init__.py:6]: I resolved the conflict here.

**Comment by min-xu-ai** [comment]
image

It is interesting that at some point the release list stopped having the msgs about what's add and what's fixed. Do you know why?

**Comment by blefaudeux** [comment]
> image
> 
> It is interesting that at some point the release list stopped having the msgs about what's add and what's fixed. Do you know why?

hmm no, good question ! I'll have a look

File: fairscale/__init__.py (hunk 1 above)
Comment (blefaudeux) [reply fairscale/__init__.py:6]: thanks !

**Pull request merged**